setup.py

```diff
@@ -27,6 +27,7 @@
 ################################################################################
 # Programmatically generate some extras combos.
 ################################################################################
+setups = read_configuration("setup.cfg")['options']['setup_requires']
```

Comment:
We shouldn't need this, it should be auto-read from the config. (We only read in extras because we modify it)

Reply:
Got it.